src/hapi_schema/db_conflict_event.py

```diff
@@ -0,0 +1,95 @@
+"""ConflictEvent table and view."""
+
+from datetime import datetime
+
+from sqlalchemy import (
+    DateTime,
+    Enum,
+    ForeignKey,
+    Integer,
+    select,
+)
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from hapi_schema.db_admin1 import DBAdmin1, DBLocation
+from hapi_schema.db_admin2 import DBAdmin2
+from hapi_schema.utils.base import Base
+from hapi_schema.utils.constraints import (
+    population_constraint,
+    reference_period_constraint,
+)
+from hapi_schema.utils.enums import EventType
+from hapi_schema.utils.view_params import ViewParams
+
+
+# normalised table
+class DBConflictEvent(Base):
+    __tablename__ = "conflict_event"
+    __table_args__ = (
+        population_constraint(
+            population_var_name="events"
+        ),  # not really a population
```

Comment:
Should the constraint be renamed to "positive_constraint" - maybe the default var name can still be "population"?

Reply:
Perhaps, but we can clean that up later. Probably best to stick with "population" for now.